operations/research-and-development/product/product-planning/how-to-use-productboard.md

```diff
@@ -96,7 +97,7 @@ Two very important notes regarding this integration:
 * Do reorder your features and play around with scoring your features.
 * Do add sub-components if you feel they are appropriate and do not exist anywhere else.
 * Do create your own feature views with custom prioritization scores (using existing drivers).
-* Do post requests for all of the following in the ProductBoard channel.
+* Do post requests for all of the following in the productboard channel.
```

Comment:
Suggest adding a link to the productboard channel if possible.

Reply:
Good idea. Here is the link:
https://community.mattermost.com/core/channels/pm-feature-request-and-roadmap-tool